providers/BaseProviders/src/main/java/com/palmergames/bukkit/towny/scheduling/TaskScheduler.java

```diff
@@ -21,33 +21,21 @@ public interface TaskScheduler {
 
 	ScheduledTask run(Consumer<ScheduledTask> task);
 
-	default ScheduledTask run(Entity entity, Consumer<ScheduledTask> task) {
```

Comment:
These defaults can be kept to avoid more diff in the bukkit scheduler

Reply:
My idea is to separate the scheduler of folia and the scheduler of bukkit to avoid confusion. If you think this is inappropriate, you can change it on my branch.

Reply:
Sorry, my bad

Reply:
sorry my bad